tests/components/unifi/test_services.py

```diff
@@ -1,58 +1,39 @@
 """deCONZ service tests."""
 
-from unittest.mock import Mock, patch
+from unittest.mock import patch
 
 from homeassistant.components.unifi.const import DOMAIN as UNIFI_DOMAIN
-from homeassistant.components.unifi.services import (
-    SERVICE_REMOVE_CLIENTS,
-    UNIFI_SERVICES,
-    async_setup_services,
-    async_unload_services,
-)
+from homeassistant.components.unifi.services import SERVICE_REMOVE_CLIENTS
 
 from .test_controller import setup_unifi_integration
 
 
-async def test_service_setup(hass):
+@patch("homeassistant.components.unifi.async_unload_services")
+@patch("homeassistant.components.unifi.async_setup_services")
```

Comment:
I liked the previous way of patching since it doesn't depend on integration details.

Reply:
Fixed